tests/istio_testcases/example_policies/bookinfo-demo/bookinfo-policy-test-unlimited-attrs.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: details-viewer-add
```

Comment:
consider renaming this, as not all operation components here are with the 'add' semantics.